src/context/WalletProvider/KeepKey/helpers.ts

```diff
@@ -60,3 +64,17 @@ export const parseIntToEntropy = (entropy: string): RecoverDevice['entropy'] =>
   const parsedEntropy = Math.floor(Number(entropy))
   return isValidEntropyNumber(parsedEntropy) ? parsedEntropy : VALID_ENTROPY_NUMBERS[0]
 }
+
+export const getPlatform = () => {
```

Comment:
this is the kinda stuff we should use libraries for, getting platform detection correct is notoriously flaky

Reply:
Can't find any recently updated libs for platform detection, mostly just references to using the Navigator Web API.